lib/analytics.js

```diff
@@ -200,15 +203,24 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(
   // initialize integrations, passing ready
   // create a list of any integrations that did not initialize - this will be passed with all events for replay support:
   this.failedInitializations = [];
+  var initialPageSkipped = false;
   each(function(integration) {
     if (
       options.initialPageview &&
       integration.options.initialPageview === false
     ) {
-      integration.page = after(2, integration.page);
+      // We've assumed one initial pageview, so make sure we don't count the first page call.
+      var page = integration.page;
+      integration.page = function() {
+        if (initialPageSkipped) {
+          return page.apply(this, arguments);
+        }
+        return (initialPageSkipped = true);
```

Comment:
what's the expected turn value for this method? If it's a boolean, I think you want:

```
return (initialPageSkipped === true);
```

Reply:
fixed!